cpp/src/barretenberg/honk/flavor/ultra.hpp

```diff
@@ -41,6 +50,25 @@ class Ultra {
     // The total number of witness entities not including shifts.
     static constexpr size_t NUM_WITNESS_ENTITIES = 11;
 
+    // define the tuple of Relations that comprise the Sumcheck relation
+    using Relations = std::tuple<sumcheck::UltraArithmeticRelation<FF>,
+                                 sumcheck::UltraArithmeticRelationSecondary<FF>,
+                                 sumcheck::UltraGrandProductComputationRelation<FF>,
+                                 sumcheck::UltraGrandProductInitializationRelation<FF>,
+                                 sumcheck::LookupGrandProductComputationRelation<FF>,
+                                 sumcheck::LookupGrandProductInitializationRelation<FF>,
+                                 sumcheck::GenPermSortRelation<FF>,
+                                 sumcheck::EllipticRelation<FF>,
+                                 sumcheck::AuxiliaryRelation<FF>>;
+
+    static constexpr size_t MAX_RELATION_LENGTH = get_max_relation_length<Relations>();
+    static constexpr size_t NUM_RELATIONS = std::tuple_size<Relations>::value;
+
+    // define the container for storing the univariate contribution from each relation in Sumcheck
+    using UnivariateTuple = decltype(create_univariate_tuple<FF, Relations, 0>());
```

Comment:
Can you just write `create_univariate_tuple<FF, Relations, 0>` since you're using the default value `0`? Then I don't have to wonder what that parameter is. Ditto elsewhere.

Reply:
Updated here and in standard